probe/data.go

```diff
@@ -152,6 +162,8 @@ func LoadDataFromFile(filename string) error {
 					log.Debugf("Load meta data: name[%s], version[%s]", metaData.Name, metaData.Ver)
 				}
 			} else {
+				mutex.Lock()
+				defer mutex.Unlock()
```

Comment:
propose not to use `defer` in a loop.

Reply:
I think I can remove this lock, as the LoadDataFromFile() only be called when the EaseProbe start.